### DISCUSSION TITLE
How to set default behaviour to deselect 'Registry items' from Junk/Leftover items after Uninstallation is successful?

### DISCUSSION THREAD

**Discussion by ChaliceChore** [Klocman/Bulk-Crap-Uninstaller#588] [Q&A] [answered]
Pretty much the title.
From what I've learnt wandering the internet over the years, clearing up unused registry entries is NOT at all recommended. So, I don't want to touch them after the uninstallation is done. I want the checkbox for all registry items to be deselected by default. However, I do want the remaining items selected. How do I accomplish this?

**Comment by Klocman** [comment, marked as answer]
Currently there is no way to do this, and while it is true that you should not delete things haphazardly, it applies to files just as much as to registry keys.

I suggest turning leftover detection off if you're worried about removing something important, as in almost all cases it will only free up minimal amounts of space with no effect on system speed.

**Reply by ChaliceChore** [reply]
Ok I understand.
